debian/control

```diff
@@ -4,8 +4,9 @@ Priority: optional
 Homepage: http://ceph.com/
 Vcs-Git: git://github.com/ceph/ceph.git
 Vcs-Browser: https://github.com/ceph/ceph
-Maintainer: Laszlo Boszormenyi (GCS) <gcs@debian.hu>
-Uploaders: Sage Weil <sage@newdream.net>
+Maintainer: Ceph Maintainers <ceph-maintainers@lists.ceph.com>
```

Comment:
@dachary But does Laszlo have access to Jenkins and our VPN to upload packages to ceph.com?